tvtk/pyface/ui/qt4/decorated_scene.py

```diff
@@ -158,6 +158,7 @@ def _save_snapshot(self):
                             "Geomview", "VRML", "Wavefront", "Open Inventor",
                             "Povray", "X3D"]
 
+            wildcard = ""
```

Comment:
I agree that accessing `_save_snapshot` directory would be a smell, but I  don't think it's necessary: you should be able to call `perform` on the related `Action` object instead. You'd have to write logic to retrieve the relevant action in the first place from `DecoratedScene.actions`, and I'd suggest adding an `id` to the individual actions to make that easier.